content/en/docs/concepts/scheduling-eviction/assign-pod-node.md

```diff
@@ -124,8 +124,8 @@ For example, consider the following Pod spec:
 
 In this example, the following rules apply:
 
-  * The node *must* have a label with the key `topology.kubernetes.io/zone` and
-    the value of that label *must* be either `antarctica-east1` or `antarctica-west1`.
+  * The node *must* have a label with the key `kubernetes.io/os` and
```

Comment:
This doesn't look right to me. The yaml file directly preceding this text ([pod-with-node-affinity.yaml](https://raw.githubusercontent.com/kubernetes/website/main/content/en/examples/pods/pod-with-node-affinity.yaml)) doesn't include the `kubernetes.io/os` label. Perhaps I'm missing something.